sidebars.js

```diff
@@ -269,9 +269,93 @@ module.exports = {
         }
       ]
     },
-    { type: 'doc', id: 'reference/core-concepts' },
-    { type: 'doc', id: 'community' },
-    { type: 'doc', id: 'contributing/contributing' },
-    { type: 'doc', id: 'changelog' }
+    {
+      type: 'category',
+      label: 'Reference',
+      collapsed: true,
+      items: [
+        {
+          type: 'category',
+          label: 'Glossary of Expectations',
+          items: [
+            { type: 'doc', id: 'reference/core-concepts' }
+          ]
+        },
+        {
+          type: 'category',
+          label: 'Core Concepts',
+          items: [
+            { type: 'doc', id: 'reference/core-concepts' },
+            { type: 'doc', id: 'reference/checkpoints-and-actions' },
+            { type: 'doc', id: 'reference/data-context' },
+            { type: 'doc', id: 'reference/data-discovery' },
+            { type: 'doc', id: 'reference/data-docs' },
+            { type: 'doc', id: 'reference/datasources' },
+            { type: 'doc', id: 'reference/evaluation-parameters' },
+            { type: 'doc', id: 'reference/expectations/conditional-expectations' },
+            { type: 'doc', id: 'reference/expectations/distributional-expectations' },
+            { type: 'doc', id: 'reference/expectations/expectations' },
+            { type: 'doc', id: 'reference/expectations/implemented-expectations' },
+            { type: 'doc', id: 'reference/expectation-suite-operations' },
+            { type: 'doc', id: 'reference/execution-engine' },
+            { type: 'doc', id: 'reference/metrics' },
+            { type: 'doc', id: 'reference/profilers' },
+            { type: 'doc', id: 'reference/expectations/result-format' },
+            { type: 'doc', id: 'reference/expectations/standard-arguments' },
+            { type: 'doc', id: 'reference/stores' },
+            { type: 'doc', id: 'reference/dividing-data-assets-into-batches' },
+            { type: 'doc', id: 'reference/validation' }
+          ]
+        },
+        {
+          type: 'category',
+          label: 'Supporting Resouces',
+          items: [
+            { type: 'doc', id: 'reference/core-concepts' }
+          ]
+        },
+        {
+          type: 'category',
+          label: 'Spare Parts',
+          collapsed: true,
+          items: [
+            { type: 'doc', id: 'reference/core-concepts' }
+           ]
+        },
+        {
+          type: 'category',
+          label: 'API Reference',
+          collapsed: true,
+          items: [
+            { type: 'doc', id: 'reference/core-concepts' }
+           ]
+       }
+     ]
```

Comment:
Currently these docs refer to the same `reference/core-concepts` which seems to open up all the dropdown menus when clicked. It would be nice to have individual stubs for the categories like `Spare Parts` and `API Reference` so they can be displayed and written separately

Reply:
Adding in a stub